### PR TITLE
add user query log

### PR DESCRIPTION
**Pull request by maxis1718**
record `user_name`, `user_input` and `date` in `beauty.query` colloection

### CODE REVIEW

**Comment by maxis1718** [comment]
:rocket:

File: lib/beauty.js [hunk 1]
```diff
@@ -32,6 +33,13 @@ function extract_tag (text) {
     return null;
 }
 
+function extract_human_language (text) {
+    // human language input
+    // 布萊恩 給我100讚的大眼妹
```
Comment (bryanyuan2) [inline lib/beauty.js:38]: Please do not use inappropriate term as comment

Comment (maxis1718) [inline lib/beauty.js:38]: won't fix

Comment (ctweng) [inline lib/beauty.js:38]: This comment is needed. Do not remove it.

Comment (conradhuang) [inline lib/beauty.js:38]: For the justice, please keep the comment as it is

**Pull request merged**